core/src/main/java/org/apache/iceberg/encryption/EncryptionUtil.java

```diff
@@ -71,30 +70,35 @@ public static KeyManagementClient createKmsClient(Map<String, String> catalogPro
   }
 
   public static EncryptionManager createEncryptionManager(
-      Map<String, String> tableProperties, KeyManagementClient kmsClient) {
+      String tableKeyId, int dekLength, KeyManagementClient kmsClient) {
```

Comment:
I'm fine with adding a signature here, but this should not add breaking changes to revapi.

Can you separate this into two methods and keep the old signature?